docs/core_concepts/0_draft_and_deploy/index.md

```diff
@@ -0,0 +1,39 @@
+# Draft and Deploy
+
+Windmill lets you to iterate on scripts, flows and apps before deploying them.
+
+Each script, flow or app can go through 3 states: local edit, draft and deployed.
+
+<video
+    className="border-2 rounded-xl object-cover w-full h-full"
+    autoPlay
+    controls
+    id="main-video"
+    src="/videos/draft_and_deploy.mp4"
+/>
+
+## Local Edits
+
+All changes on a runnable (script, flow or app) are only on your browser until validated "Draft" or "Deploy". Until then, they are only visible locally.
+
+Local edits can be run and tested only from the editor (script, flow or app) with the `Test`button.
+
+## Draft
+
+Local edits can be saved as Draft as a staging state. A draft is visible from members of the workspace and can be edited. There is only one draft per runnable.
```

Comment:
The draft inherits the permission of the item it is attached to or at the path they are in for drafts non deployed items.